crates/store/src/db/models/queries/accounts.rs

```diff
@@ -148,6 +149,79 @@ pub(crate) fn select_accounts_by_id(
     Ok(account_infos)
 }
 
+pub(crate) fn select_account_vault_assets(
+    conn: &mut SqliteConnection,
+    account_id: AccountId,
+    block_from: BlockNumber,
+    block_to: BlockNumber,
+) -> Result<(BlockNumber, Vec<AccountVaultValue>), DatabaseError> {
+    use schema::account_vault_assets as t;
+
+    // SELECT block_num, faucet_id, delta
+    // FROM account_fungible_asset_deltas
+    // WHERE account_id = ?
+    //   AND block_num >= ?
+    //   AND block_num <= ?
+    // ORDER BY block_num ASC, faucet_id ASC
+    // LIMIT ROW_LIMIT;
```

Comment:
nit: I'd format this as:
```rust
// SELECT
//     block_num,
//     faucet_id,
//     delta
// FROM
//     account_fungible_asset_deltas
// WHERE
//     account_id = ?
//     AND block_num >= ?
//     AND block_num <= ?
// ORDER BY
//     block_num ASC,
//     faucet_id ASC
// LIMIT
//     ROW_LIMIT;
```
Also, I think this query is not correct - shouldn't we use `account_vault_assets` for this?